test/__fixtures__/ajv.js

```diff
@@ -2,7 +2,7 @@ const Ajv = require('ajv');
 const addFormats = require('ajv-formats');
 
 const schemas = {
-  "http://identity.foundation/claim-format-registry/schemas/presentation-definition-claim-format-designations.json": {
+  "https://identity.foundation/claim-format-registry/schemas/presentation-definition-claim-format-designations.json": {
     "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Thanks for this PR we will merge it as is, but will fork of another PR that generally looks at all the occurrences of http vs https. (like the one here)

Reply:
@kimdhamilton Seems like all the `http://identity.foundation` (apart from in package.json) where addressed in this PR. So not sure about this comment anymore.